apps/bors_frontend/web/static/js/time-convert.js

```diff
@@ -0,0 +1,6 @@
+const items = document.querySelectorAll(".time-convert");
+Array.prototype.forEach.call(items, function(time) {
+    const date = new Date(time.innerHTML);
```

Comment:
what's the reason you are doing this conversion in js instead of in elixir?

Reply:
Because it means it can grab the timezone of the browser itself. Doing it in Elixir would require bors to have a user setting for the preferred timezone stored in the database or something.